internal/connect/command_custom_plugin_version_create.go

```diff
@@ -0,0 +1,146 @@
+package connect
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	connectcustompluginv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/utils"
+)
+
+type pluginVersionOut struct {
+	Version             string   `human:"Version" serialized:"version"`
+	VersionNumber       string   `human:"Version Number" serialized:"version_number"`
+	IsBeta              string   `human:"Beta" serialized:"is_beta"`
+	ReleaseNotes        string   `human:"Release Notes" serialized:"release_notes"`
+	SensitiveProperties []string `human:"Sensitive Properties" serialized:"sensitive_properties"`
+}
+
+func (c *customPluginCommand) newVersionCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a custom connector plugin version.",
+		Args:  cobra.NoArgs,
+		RunE:  c.createCustomPluginVersion,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Create custom connector plugin version for plugin "ccp-123456".`,
+				Code: "confluent connect custom-plugin version create --plugin ccp-123456 --plugin-file datagen.zip --version-number 0.0.1 --beta=false --sensitive-properties=property1,property2",
```

Comment:
`--beta=false` is the default, right? I don't think you would ever do this.
```suggestion
				Code: "confluent connect custom-plugin version create --plugin ccp-123456 --plugin-file datagen.zip --version-number 0.0.1 --beta=true --sensitive-properties=property1,property2",
```